pkg/service/model_build_load_balancer.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"net"
 	"regexp"
+	"sigs.k8s.io/aws-load-balancer-controller/pkg/algorithm"
```

Comment:
nit: lets group this with the local imports

Reply:
👍